From 3306650e1332c9a5ed615d3970fc9d51367cd792 Mon Sep 17 00:00:00 2001
From: Yarsin He <yarsin.he@citrix.com>
Date: Tue, 9 Jan 2018 17:26:22 +0800
Subject: [PATCH] Update due to upstream core_kernel updates to 0.10.0

Signed-off-by: Yarsin He <yarsin.he@citrix.com>
---
 main.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/main.ml b/main.ml
index dbe683a..fc63c61 100644
--- a/main.ml
+++ b/main.ml
@@ -579,7 +579,7 @@ let process root_dir name x =
       match List.Assoc.find x.Storage.Volume.Types.keys _clone_on_boot_key with
       | None -> set
       | Some transient -> Set.add set transient
-    ) ~init:(Set.empty ~comparator:String.comparator) response in
+    ) ~init:(Core.String.Set.empty) response in
     let response = List.filter ~f:(fun x -> not(Set.mem transients x.Storage.Volume.Types.key)) response in
     let response = List.map ~f:vdi_of_volume response in
     Deferred.Result.return (R.success (Args.SR.Scan.rpc_of_response response))
-- 
2.12.2.windows.2

