From 31fd00b22921d32d1c2dd42e18a53dfe8cfb18b1 Mon Sep 17 00:00:00 2001
From: Gabor Igloi <gabor.igloi@citrix.com>
Date: Thu, 19 Oct 2017 17:09:08 +0100
Subject: [PATCH] lwt/jbuild: update dependencies to Lwt 3

Signed-off-by: Gabor Igloi <gabor.igloi@citrix.com>
---
 lwt/jbuild | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lwt/jbuild b/lwt/jbuild
index fa0b657..c02a0eb 100644
--- a/lwt/jbuild
+++ b/lwt/jbuild
@@ -5,7 +5,7 @@
    (cohttp
     cstruct
     lwt
-    lwt.ssl
+    lwt_ssl
     lwt.unix
     re.str
     rpclib
-- 
2.7.4

