From 4a6e134ff8325c3cb8fc298bcedb98c8ece70479 Mon Sep 17 00:00:00 2001
From: Marcello Seri <marcello.seri@citrix.com>
Date: Wed, 11 Apr 2018 14:14:30 +0100
Subject: [PATCH 2/2] update opam and _oasis file for the lwt/lwt_log splitup

Signed-off-by: Marcello Seri <marcello.seri@citrix.com>
---
 _oasis | 4 ++--
 opam   | 1 +
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/_oasis b/_oasis
index 4b63ecc..14f81db 100644
--- a/_oasis
+++ b/_oasis
@@ -27,7 +27,7 @@ Executable "shared-block-ring"
   MainIs:             main.ml
   Custom:             true
   Install:            false
-  BuildDepends:       lwt, lwt.unix, mirage-block-unix, shared-block-ring, cstruct, oUnit, io-page, io-page-unix, cmdliner
+  BuildDepends:       lwt, lwt.unix, lwt_log, mirage-block-unix, shared-block-ring, cstruct, oUnit, io-page, io-page-unix, cmdliner
 
 Executable test
   CompiledObject:     best
@@ -35,7 +35,7 @@ Executable test
   MainIs:             test.ml
   Custom:             true
   Install:            false
-  BuildDepends:       lwt, lwt.unix, mirage-block-unix, mirage-clock-unix, shared-block-ring, cstruct, oUnit, io-page, io-page-unix
+  BuildDepends:       lwt, lwt.unix, lwt_log, mirage-block-unix, mirage-clock-unix, shared-block-ring, cstruct, oUnit, io-page, io-page-unix
 
 Test test
   Command:            ./test.native
diff --git a/opam b/opam
index 6da1909..386ad1d 100644
--- a/opam
+++ b/opam
@@ -21,6 +21,7 @@ depends: [
   "ppx_deriving" {build}
   "ppx_sexp_conv" {build}
   "lwt"
+  "lwt_log"
   "ocamlfind"
   "ounit"
   "mirage-types-lwt" {>= "3.0.0"}
-- 
2.14.3

