From 5c66e9de9a39a322082ccbf0734b03854ff02624 Mon Sep 17 00:00:00 2001
From: Marcello Seri <marcello.seri@citrix.com>
Date: Tue, 16 May 2017 18:00:47 +0100
Subject: [PATCH] Add cmx files among the installed ones

Signed-off-by: Marcello Seri <marcello.seri@citrix.com>
---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 1c15162..ca17eaa 100644
--- a/Makefile
+++ b/Makefile
@@ -48,7 +48,7 @@ export OCAMLDEP
 all: byte native
 
 byte: oclock.cma dlloclock.so
-native: oclock.cmxa liboclock.a oclock.a
+native: oclock.cmx oclock.cmxa liboclock.a oclock.a
 
 # Generic library compilation
 %.cma: %.cmo dll%.so
@@ -87,10 +87,10 @@ distclean: clean
 # (Un)Install
 install: all
 ifdef OCAMLFIND
-	$(OCAMLFIND) install oclock oclock.cma oclock.cmxa liboclock.a oclock.cmi oclock.a META -dll dlloclock.so $(OCAMLFIND_DESTDIR)
+	$(OCAMLFIND) install oclock oclock.cma oclock.cmxa oclock.cmx liboclock.a oclock.cmi oclock.a META -dll dlloclock.so $(OCAMLFIND_DESTDIR)
 else
 	install -d $(DESTDIR)$(INSTALL_DIR)
-	install -t $(DESTDIR)$(INSTALL_DIR) oclock.cma oclock.cmxa liboclock.a oclock.cmi oclock.a META
+	install -t $(DESTDIR)$(INSTALL_DIR) oclock.cma oclock.cmxa oclock.cmx liboclock.a oclock.cmi oclock.a META
 	install -t $(DESTDIR)$(STUBLIBS_DIR) dlloclock.so
 endif
 
-- 
2.11.0

