From ae3c35f7f3ba50c12a2dd720b6fde98d800e2fe7 Mon Sep 17 00:00:00 2001
From: Gabor Igloi <gabor.igloi@citrix.com>
Date: Fri, 20 Oct 2017 17:43:11 +0100
Subject: [PATCH] opam: remove async bound

Signed-off-by: Gabor Igloi <gabor.igloi@citrix.com>
---
 opam | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/opam b/opam
index f88a4fe..47451b4 100644
--- a/opam
+++ b/opam
@@ -31,6 +31,6 @@ depends: [
   "mirage-block-unix" {>= "2.4.0"}
   "shared-block-ring" {>= "2.3.0"}
   "cmdliner"
-  "async" {< "v0.9.0"}
+  "async"
 ]
 
-- 
2.7.4

