From 7ef2ab8cdb06c460cbc40f838982116d2f5a3056 Mon Sep 17 00:00:00 2001
From: Gabor Igloi <gabor.igloi@citrix.com>
Date: Fri, 20 Oct 2017 23:23:05 +0100
Subject: [PATCH 07/13] Switch to nanoseconds used by new shared-block-ring
 interface

Signed-off-by: Gabor Igloi <gabor.igloi@citrix.com>
Signed-off-by: Marcello Seri <marcello.seri@citrix.com>
---
 switch/switch_main.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/switch/switch_main.ml b/switch/switch_main.ml
index 845b103..53d3559 100644
--- a/switch/switch_main.ml
+++ b/switch/switch_main.ml
@@ -173,7 +173,7 @@ let make_server config =
       Block.connect ~buffered:true redo_log_path
       >>= fun block ->
       let open Lwt_result in
-      Redo_log.start ~flush_interval:5. block (process_redo_log statedir)
+      Redo_log.start ~flush_interval:5_000_000_000L block (process_redo_log statedir)
       >>= fun redo_log ->
       info "Redo-log playback complete: everything should be in sync";
       queues := !on_disk_queues; (* everything should be in sync *)
-- 
2.14.1

